index.html

```diff
@@ -146,9 +146,9 @@ <h2>Tonic</h2>
             </section>
         </section>
 
-        <section class="project-section">
+        <section class="project-section" id="professional-art-card">
             <figure>
-                <img src="./Images/Snapshoot Portfolio (2).png" alt="" id="snap-shot-image">
+                <img src="./Images/Snapshoot Portfolio (2).png" alt="" class="snap-shot-image">
             </figure>
             <section class="project-showing">
                 <h2>Multi-Post
```

Comment:
- [ ] While you make a great effort 😃 I see that you need to increase the margin between the sections from the top and bottom sides as you see here it is less than the Figms, so please 💯😊 try to increase the margin to make it nice like the Figma 🚀😊
- [ ] Also consider 😃 increasing the margin between the sections from the top and bottom sides at the mobile version as you see the distance between the sections is not like the Figma, so let's try to increase it like the Figma to make it nice 👌😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

> In the desktop version 😃🚀

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/235676774-4b4824bd-422f-418d-945e-0402b3d3f458.png)   |    ![image](https://user-images.githubusercontent.com/83360636/235676697-2bf04789-bbf8-4cd7-b547-be7c4f506f46.png)   |

> In the mobile version 👌🚀

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/235685007-a6827e79-f21a-46a2-9b68-972360d878fe.png)  |    ![image](https://user-images.githubusercontent.com/83360636/235685095-9b76a3a7-fdfb-4839-94fa-7e43cd70f2e3.png)   |